.github/workflows/publish.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and upload to PyPI
+
+on: [push, pull_request]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+      # We will need cibuildwheel if we add a cython extension for common beam calcs
+      # - name: Install cibuildwheel
+      #  run: python -m pip install cibuildwheel
```

Comment:
Maybe remove these last two commented lines because that's not the only thing that will change